simpleflow/logging_context.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+
+ENV_KEYS = {
+    "activity_id": "_SWF_CONTEXT_ACTIVITY_ID",
```

Comment:
(Not too serious) Maybe `"activity_id"` et al. should be constants :stuck_out_tongue: